.github/workflows/ci.yml

```
@@ -157,6 +157,7 @@ jobs:
             export NIMFLAGS="-d:nimRawSetjmp"
           fi
           nim --version
+          nimble install "nimble>=0.14.2"
```

This should probably fix the CI failure:

```suggestion
          nimble install -y "nimble>=0.14.2"
```